content/03.prior-knowledge.cascadelearning.md

```diff
@@ -0,0 +1,31 @@
+## Cascade Learning {.page_break_before}
+
+Sometimes, due to paucity of data and its unstable predictive capabilities, one may feel the need to consult various machine learning methods and their predictions to make an educated decision about the identified predictive patterns, collectively called _ensemble learning_. 
+In an ensemble learning workflow, one may use various parameters to maximize the learned information. 
+A consensus approach is often ideal to select predictive features from a group of independently running models. 
+At other times, one may coordinate various learning methods in such a way that one model uses information from another model in augmenting its predictive capability. 
+Random forest algorithms are a great example of ensemble learning where a consensus of the features selected through many different and independent decision trees are considered and ranked to make a final ranked list of important features that are predictive of a certain outcome (e.g. disease) [@doi:10.1186/1472-6947-13-134, @doi:10.1023/A:1010933404324].
+For the ensemble of decision trees, e.g. random forest, bootstrap aggregation (or _bagging_) helps in forming the consensus[@doi:10.1214/aos/1031689014, @doi:10.1177/2045894019890549, @doi:10.1016/S0031-3203(02)00169-3].
+However, successful application of consensus based ensemble learning algorithms requires what is known as "gold standard" data. 
+These generally refer to well labeled cases where the diagnosis or label of a data point in the training dataset has very little uncertainty (or "label-noise") associated with it [@doi:10.1093/jamia/ocw028]. 
+In most cases of rare disease, due to the inherent nature of being less defined, the symptoms as well as any underlying biology comes with a reasonable amount of uncertainty (or "label-noise") leading to a _silver standard_ dataset[@doi:10.1109/TNNLS.2013.2292894, @doi:10.1093/jamia/ocw028].
+This kind of data begets modifications in ensemble learning methods where one method leverages findings from another method to capture stable patterns existing in the _silver standard_ data and reduce uncertainty.
+Such methods fall into the category of cascade learning.
```

Comment:
Title is cascade learning, but cascade learning doesn't come up until here. Perhaps better to use ensemble learning as the title? Or separate into its own section.

Reply:
:) Yes.. this started off as a section for cascade learning, but based on @allaway and @jaclyn-taroni 's reviews and the clear need for introducing ensemble learning before delving into cascade learning, I have been adding more ensemble learning introduction to this PR. But missed adding this information to the PR description. Apologies for the confusion!